contrib/ccc_poc_env.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+export http_proxy=http://gateway:3128
+export https_proxy=${http_proxy}
+export ftp_proxy=${http_proxy}
+export GALAXY_RUN_ALL=1
+#Creating env variables for location of exe/jar files, to be used by xml/py files.
+GENOMICS_DIR=/opt/sync/Genomics
+CCC_SYNC_DIR=/cluster_share/ccc_sync
+PYTHON_2_7_PATH=$CCC_SYNC_DIR/tools/galaxy/venv
+source $PYTHON_2_7_PATH/bin/activate
+export BWA_DIR=$GENOMICS_DIR/ohsu/dnapipeline/bwa-0.7.4
+export BWA_EXE_PATH=$BWA_DIR/bwa
+export SAMTOOLS_DIR=$GENOMICS_DIR/ohsu/dnapipeline/samtools-0.1.19
+export SAMTOOLS_EXE_PATH=$SAMTOOLS_DIR/samtools
+export SAMTOOLS_v020_EXE_PATH=$GENOMICS_DIR/ohsu/dnapipeline/samtools/samtools/samtools
+export PICARD_PATH=$GENOMICS_DIR/ohsu/dnapipeline/picard-tools-1.110
+export GATK_JAR_PATH=$GENOMICS_DIR/GATK-3.2/target/GenomeAnalysisTK.jar
+#FIXME
+export CP_PATH=$CCC_SYNC_DIR/tools/cellprofiler/cp_pipeline
+export LABKEY_USERNAME=ccc@ccc.com
+export LABKEY_PASSWORD=Intel123
+export OMERO_IMAGE_SELECTION=$CCC_SYNC_DIR/tools/omero
+#END FIXME
+export SEATTLESEQ_GETANNOTATION_JAR_PATH=$GENOMICS_DIR/annotation/getAnnotation/project/
+export SEATTLESEQ_WRITEGENOTYPE_JAR_PATH=$GENOMICS_DIR/annotation/writeGenotype/project/
+export MUTECT_JAR_PATH=$GENOMICS_DIR/ohsu/dnapipeline/mutect-1.1.7.jar
+export BOWTIE_TOOLS_EXE_PATH=$GENOMICS_DIR/ohsu/rnapipeline/bowtie2-2.1.0/
+export TOPHAT_EXE_PATH=$GENOMICS_DIR/ohsu/rnapipeline/tophat-2.0.14/
+export CUFFLINKS_TOOLS_EXE_PATH=$GENOMICS_DIR/ohsu/rnapipeline/cufflinks-2.2.1/
+export CUFFDIFF_SCRIPT_PATH=$GENOMICS_DIR/ohsu/rnapipeline
+export SNPEFF_JAR_PATH=$GENOMICS_DIR/annotation/snpEff/
+export ONCOTATOR_PATH=$CCC_SYNC_DIR/tools/Oncotator
```

Comment:
Yes, I had accessibility issues from within the virt_test_env/bin (even when it was sourced). I guess adding the source /cluster_share/ccc_sync/tools/Oncotator/oncotator-1.5.1.0/virt_test_env/bin/activate before this variable will solve the problem when the job is submitted with condor. I can test this in the new cluster. If it doesn't work, then we can move the binary to the root directory. Sorry, i missed noting it down in the README.